core-services/prow/02_config/_plugins.yaml

```diff
@@ -720,6 +720,12 @@ config_updater:
         - ci
       gzip: true
       name: ci-operator-misc-configs
+    ci-operator/config/open-cluster-management/**/*-release-2.5*.yaml:
```

Comment:
Do we have any files satisfying this?

Reply:
I wonder how it passed our check if they exist.

Reply:
Not yet. They can't be added until the _plugins.yaml is updated.
https://github.com/openshift/release/pull/23147